emulate/emulate_queue.py

```diff
@@ -180,7 +183,12 @@ def main():
 			logging.info(f'Running code for {animation.duration_of_emulation_in_seconds} seconds')
 			logging.info(f'Animation name: {animation.name}')
 
-			run_code(animation.code, animation.duration_of_emulation_in_seconds)
+			return_code = run_code(animation.code, animation.duration_of_emulation_in_seconds)
```

Comment:
Ogólnie, to zwracany typ run_code za bardzo nam nic nie mówi. W języku C można spotkać, że są return codes itp. 
Nie ma to super dużego znaczenia chyba w tym przypadku, ale zastanawiam się czy nie wprowadzić naszego custom wyjątku tutaj i zamiast sprawdzać co zwaraca, to łapać nasz wyjątek.

Reply:
Może rzeczywiście to będzie bardziej pythonowe podejście